emily/handler/src/database/accessors.rs

```diff
@@ -75,6 +85,66 @@ pub async fn get_deposit_entries_for_transaction(
     .await
 }
 
+/// Updates a deposit.
+pub async fn update_deposit(
+    context: &EmilyContext,
+    update: &DepositUpdatePackage,
+) -> Result<DepositEntry, Error> {
+    // Setup the update procedure.
+    let update_expression: &str = " SET
+        #history = list_append(#history, :new_event),
+        #version = #version + :one,
+        #op_status = :new_op_status,
+        #height = :new_height,
+        #hash = :new_hash
+    ";
+    // Ensure the version field is what we expect it to be.
+    let condition_expression = "attribute_exists(#version) AND #version = :expected_version";
+    // Make the key item.
+    let key_item: Item = serde_dynamo::to_item(&update.key)?;
+    // Get simplified status enum.
+    let status: Status = (&update.event.status).into();
+    // Build the update.
+    context
+        .dynamodb_client
+        .update_item()
+        .table_name(&context.settings.deposit_table_name)
+        .set_key(Some(key_item.into()))
+        .expression_attribute_names("#history", "History")
+        .expression_attribute_names("#version", "Version")
+        .expression_attribute_names("#op_status", "OpStatus")
+        .expression_attribute_names("#height", "LastUpdateHeight")
+        .expression_attribute_names("#hash", "LastUpdateBlockHash")
```

Comment:
Curious, why use these aliases?

Reply:
The examples online did, but I just did a test and it seems like you don't need to use aliases. That said you DO need to use an alias for any literal you plan on adding in (`:one` for example).

I'll alter this function to not use these aliases because it's unnecessary.

Reply:
Done!